packages/validation/src/validators/practices/rules/KBP006-high-available.ts

```diff
@@ -0,0 +1,22 @@
+import {defineRule} from '../../custom/config.js';
+import {isDeployment} from '../../custom/schemas/deployment.apps.v1.js';
+
+export const highAvailable = defineRule({
+  id: 6,
+  description: 'Require at least two replicas',
+  fullDescription: 'High availability avoids downtimes when a pod crashes.',
+  help: "Set your deployment's replicas to two or higher.",
+  advanced: {
+    enabled: false,
+    severity: 3,
+  },
+  validate({resources, params}, {report}) {
+    resources.filter(isDeployment).forEach(deployment => {
+      const replicaCount = deployment.spec?.replicas ?? 1;
+      const replicaThreshold = params ?? 1;
```

Comment:
It's `const valid = replicaCount > replicaThreshold;` so two or more is valid which is HA. 

Maybe it's a bad idea to add this rule without thinking to much about it 🤔 If we later also want to add HA for statefulsets or change it to >=3 (some would argue only that is truly highly available because it helps with quorums when one goes down) then that could be considered a breaking change since people will not be allowed to merge their CI pipeline anymore 🤔

Reply:
> It's `const valid = replicaCount > replicaThreshold;` so two or more is valid which is HA.

Yes, you are right👍 My flow of thought about this stopped on the line before this one 😓 Hmmm, but then if you configure it to `5`, would you expect that minimum of replicas is `5` or `6`? 🤔 

> Maybe it's a bad idea to add this rule without thinking to much about it 🤔 If we later also want to add HA for statefulsets or change it to >=3 (some would argue only that is truly highly available because it helps with quorums when one goes down) then that could be considered a breaking change since people will not be allowed to merge their CI pipeline anymore 🤔

Hmmm, good point. OTOH, we have configurability so it will require changing settings of this rule only. Still, changing default rules behavior should be well-thought out and communicated to users so there are no surprises.